src/Minsk/Minsk.csproj

```diff
@@ -10,7 +10,7 @@
 
   <PropertyGroup>
     <GeneratedSources>CodeAnalysis/**/*.g.cs</GeneratedSources>
-    <DefaultItemExcludes>$(GeneratedSources)</DefaultItemExcludes>
+    <DefaultItemExcludes>$(DefaultItemExcludes);$(GeneratedSources)</DefaultItemExcludes>
```

Comment:
Is this a problem? No, since the property is only present in targets. It is safe without the prepend.

The snippet from targets file for reference: https://github.com/dotnet/sdk/blob/45d5983a185706faf8d5d27f1a8ebc2d90c00116/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets#L23-L35

Reply:
IMHO it's harder to reason about. For example, I've just moved the code to `Directory.Build.props`. Replacing properties like this seems fragile to me, because you need to be certain that you don't have *any* other party that injects excludes. My brain doesn't work that way 😏